src/cli/cfm_shp_to_tsurf.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+"""
+Simple script to convert entries from the current CFM shapefile into
+multiple tsurf files.
+"""
+if __name__ == "__main__":
```

Comment:
this is valid code, but an unusual use of  `if __name__ == "__main__":`

Take a look at https://stackoverflow.com/questions/419163/what-does-if-name-main-do

You might do it at line 24, and/or you might wrap your main loop & setup in a function , then call function after the `if __name __` test. That's probably a more conventional approach.

Reply:
Hi Chris, Yes I realise this is a bit unusual, and I do understand the purpose behind the `if __name__ == "__main__":` statement.  I might rearrange things so it makes a bit more sense.